src/Microsoft.DotNet.Darc/src/Darc/Operations/AddBuildToChannelOperation.cs

```diff
@@ -255,7 +255,7 @@ private async Task<int> PromoteBuildAsync(Build build, List<Channel> targetChann
                 { "SigningValidationAdditionalParameters", _options.SigningValidationAdditionalParameters },
                 { "EnableNugetValidation", _options.DoNuGetValidation.ToString() },
                 { "EnableSourceLinkValidation", _options.DoSourcelinkValidation.ToString() },
-                { "PublishInstallersAndChecksums", _options.PublishInstallersAndChecksums.ToString() },
+                { "PublishInstallersAndChecksums", true },
```

Comment:
Does this somehow implicitly call true.ToString()?

Reply:
this syntax just calls Dictionary.Add a bunch of times, I'm surprised this works.

Reply:
It does not. This PR doesn't build

Reply:
lol, its a bool so I passed it as is.. changing it